runtime/compiler/infra/J9Cfg.cpp

```diff
@@ -1707,7 +1713,16 @@ void
 J9::CFG::propagateFrequencyInfoFromExternalProfiler(TR_ExternalProfiler *profiler)
    {
    _externalProfiler = profiler;
-
+#if defined(J9VM_OPT_MICROJIT)
+   J9Method *method = static_cast<TR_ResolvedJ9Method *>(comp()->getMethodBeingCompiled())->ramMethod();
+   U_8 *extendedFlags = reinterpret_cast<TR_J9VMBase *>(comp()->fe())->fetchMethodExtendedFlagsPointer(method);
+   if (((*extendedFlags & 0x10) != 0x10) && (uintptr_t)(method->extra) == 0x1 && TR::Options::getJITCmdLineOptions()->_mjitEnabled)
```

Comment:
Not sure how I missed this in my earlier review. I don't think you should be using `0x10` and `0x1` directly here; there's macros defined for them. At the same time it also doesn't make sense to use those macros directly here. I think the right thing to do is to create and use a front end query, one for the extended flags and one for checking if the method is interpreted; the latter query already exists.